base/server/scripts/operations

```diff
@@ -716,7 +716,18 @@ display_configuration_information()
 
 display_instance_status_systemd()
 {
+    # source the instance specific variables
+    . /etc/sysconfig/${PKI_INSTANCE_NAME}
```

Comment:
What is the scope of variables sourced within a function? If it's global we might want to do it at the beginning of the script (around line 42) like this:
```
if [ -f /etc/sysconfig/${PKI_INSTANCE_NAME} ]; then
    . /etc/sysconfig/${PKI_INSTANCE_NAME}
fi
```

Reply:
I did not see the use of the variables defined in that file except for this patch. Moreover, `/usr/share/pki/scripts/operations` is only sourced by `pkidaemon` script and it allows only `{start|status}` option. 

In order to avoid overwriting variables, I sourced it locally.

Reply:
If you're concerned about overwriting variables, you could source it in a subshell:

```
function something() {(
 if [ -f /path ]; then
  . /path
 fi 

  ..... other stuff ...
)}
```

Bute note that, other than the return code and `stdout`/`stderr`, nothing will get passed from the function back up, such as other variable modifications.

Reply:
I need a variable value. So, i guess this won't work in this case. Thanks for the suggestion, though! :)

Reply:
Sorry, to clarify you can access other variables at the time of the subshell invocation, but you can't modify their values and have the modifications persist outside of the subshell. Does that make sense?

Reply:
Yes @cipherboy. But, I think the following won't work, right?

````
#! /bin/bash
function something() {(
 if [ -f /path ]; then
  . /path  # loads the variable USE_NUXWDOG
 fi 
)}

...

something
echo "Using nuxwdog? ${USE_NUXWDOG}"
````
I need something like above. There are ways to put the above logic in place, but I wanted to keep the change as low as possible, since 10.5 is considered to be stable.

Reply:
Thanks for the discussion! I'm guessing the file is safe to source anywhere because it contains just constants. Also, we're no longer using nuxwdog in the newer code. Sorry for stirring up the trouble :)